client/src/pages/Profile/ProfileName.js

```diff
@@ -32,16 +34,30 @@ export default function ProfileName({
             variant="outlined"
             label="Name"
             xs={6}
+            // Error & HelperText do not work since name is not responsive being in a child component
```

Comment:
Can't we just pass it from parent to child component via props? Or what's the problem here?